src/shared/types.ts

```diff
@@ -3,7 +3,7 @@
  * See the LICENSE file in the repository root folder for details.
  */
 
-import { User } from '@prisma/client';
+import { User } from 'utils';
```

Comment:
good catch!

Reply:
on all of these didn't realize we did this so much

Reply:
Lmao me neither!